src/pages/ReplaceServer/ReplaceController.js

```diff
@@ -190,6 +200,18 @@ class ReplaceController extends BaseUpdateWizardPage {
       );
     }
 
+    playbook_steps.push(
+      {
+        label: translate('server.deploy.progress.monasca-rebuild'),
+        playbook: 'monasca-rebuild-pretasks.yml'
+      },
+      {
+        label: translate('server.deploy.progress.os-config'),
+        playbook: 'os-config.yml',
```

Comment:
playbook name should be osconfig-run.yml

Reply:
Thanks, good catch